mappings/net/minecraft/block/material/Material.mapping

```diff
@@ -45,6 +45,7 @@ CLASS none/ayr net/minecraft/block/material/Material
 	METHOD <init> (Lnone/ays;)V
 		ARG 0 color
 	METHOD a isFullBlock ()Z
+	METHOD b blocksLight ()Z
```

Comment:
Ok I think I like `preventsLightTransmission` and `allowsLightTransmission` so I will go with those.